web/src/views_util.ts

```diff
@@ -158,6 +158,7 @@ export function is_in_focus(): boolean {
         !modals.any_active_or_animating() &&
         !$(".home-page-input").is(":focus") &&
         !$("#search_query").is(":focus") &&
+        !$("#topic_filter_query").is(":focus") &&
```

Comment:
Probably this block should be eventually refactored to either put `.home-page-input` on the input or just select on ANY input being focused, but I figured better to merge to fix the bug immediately.